library/core/src/intrinsics.rs

```diff
@@ -984,8 +1001,19 @@ pub const unsafe fn assume(b: bool) {
 #[rustc_intrinsic]
 #[rustc_nounwind]
 #[miri::intrinsic_fallback_is_spec]
+#[inline(always)]
 pub const fn likely(b: bool) -> bool {
-    b
+    #[cfg(any(bootstrap, miri))]
```

Comment:
Without the special-case, I'm getting error in `tests/pass/shims/time-with-isolation.rs`

It reports "The loop took around 13s" instead of 12. I'm not sure why that is, because likely/unlikely are not used anywhere in the test

Reply:
The next line in that test is:
```
println!("(It's fine for this number to change when you `--bless` this test.)")
```

Reply:
So it's ok to bless the new number? I didn't want to slow things down.

Reply:
Yes, it's fine.